src/toast/ops/pipeline.py

```diff
@@ -133,22 +149,30 @@ def _exec(self, data, detectors=None, **kwargs):
                         pstr, op.name
                     )
                     log.verbose(msg)
-                    op.exec(data, detectors=selected_set)
+                    if use_acc:
```

Comment:
This pattern gets repeated. Would it not make more sense to have `op.exec(data, detectors=selected_set, use_acc=use_acc)`?

Reply:
True- the `_exec()` methods of the operators do support kwargs, so even it they are not looking for a `use_acc=False` kword, it should do no harm.

Reply:
Fixed.